src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStore.java

```diff
@@ -144,34 +153,51 @@ public BlobStoreContext getContext() throws IOException {
      */
     static boolean BREAK_CREDS;
 
+    private transient Supplier<Credentials> credentialsSupplier;
+
+    /**
+     * Make tests faster by not using CredentialsAwsGlobalConfiguration.get().sessionCredentials
+     * which can very slow especially when using EnvironmentVariableCredentialsProvider or SystemPropertyCredentialsProvider
```

Comment:
What specifically is slow without this?

Reply:
for someone leaving down under some extra network access through a VPN to us-east is more than 1s

Reply:
the idea was about caching the created session created here https://github.com/jenkinsci/aws-global-configuration-plugin/blob/bcb88b3a03bffed0153882c25c0093a627bf1851/src/main/java/io/jenkins/plugins/aws/global_configuration/CredentialsAwsGlobalConfiguration.java#L174 
but anyway with recent changes I cannot use anymore only `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`